telemetry/specs/telemetry_report.mdk

```diff
@@ -398,6 +399,45 @@ particular deployment.
   : The device local time when the packet was first received on the ingress
     physical or logical port, in nanoseconds.
 
+## Telemetry Extensions Header
+
+`
+       0                   1                   2                   3
+       0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      | NProt |        Length         |           Schema ID           |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                                                               |
+      |                                                               |
+      |                        Extensions data                        |
+      |                                                               |
+      |                                                               |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+`
+
+* NProt: Next Protocol
+  : Same definition as **NProt** in the Telemetry Report Header defined in
+    Section [#telem-report-header].
+
+* Length
+  : Indicates the length of the telemetry extensions header in multiples
+    of 4 octets.
```

Comment:
If I'm correct, the length of this extension hdr is NOT added to the length of the main report hdr. Shall we clarify that, maybe at the definition of the main hdr length field?